src/render/actor_renderer.cpp

```diff
@@ -75,23 +76,8 @@ bool isVisible( Actor const& actor, Camera const& camera )
     {
         return false;
     }
-    static std::map<int32_t, float> scaleMap;
-    auto it = scaleMap.find( actor.GetId() );
-    if( scaleMap.end() == it )
-    {
-        float& f = scaleMap[ actor.GetId() ];
-        static RenderableRepo& renderables( RenderableRepo::Get() );
-        SpriteCollection const& Sprites = renderables( actor.GetId() );
-        for( auto i = Sprites.begin(), e = Sprites.end(); i != e; ++i )
-        {
-            if( i->second->GetScale() > f )
-            {
-                f = i->second->GetScale();
-            }
-        }
-        it = scaleMap.find( actor.GetId() );
-    }
-    float scale = it->second;
+    static RenderableRepo& renderables( RenderableRepo::Get() );
```

Comment:
renderableRepo (or mRenderableRepo if static imo it's kind of okay too)
static auto& renderableRepo(RenderableRepo::Get());